src/wallet/wallet.h

```diff
@@ -941,7 +941,7 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     CPubKey vchDefaultKey;
 
     std::set<COutPoint> setLockedCoins;
-    std::set<JSOutPoint> setLockedNotes;
+    std::set<JSOutPoint> setLockedSproutNotes;
```

Comment:
The commit message incorrectly says this is a rename to `setLockedSaplingNotes`.